tailwind.config.js

```diff
@@ -0,0 +1,10 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    "./src/**/*.{js,jsx,ts,tsx}",
+  ],
+  theme: {
```

Comment:
Define the entire theme and color pallette here 
https://tailwindcss.com/docs/theme
I see A LOT of repitition in inline classname based styles which shouldn't be the case if your theming is done right and your pallete, spacing, etc. is defined right